src/renderer/components/Modal/ModalPeer.vue

```diff
@@ -147,6 +149,10 @@ export default {
 
   methods: {
     emitConnect () {
+      const host = this.form.host
+      if (host.endsWith('/')) {
+        this.form.host = host.slice(0, -1)
+      }
```

Comment:
I tested using `////` at the end:

![Screenshot from 2020-02-21 10-19-02](https://user-images.githubusercontent.com/1894191/75038267-d2fe5e00-5494-11ea-8214-9cab771a26e5.png)

You may be able to create a recursive function to remove 1 or more slashes at the end. Like this:

```
    emitConnect () {
      const { host } = this.form
      const path = url => url.endsWith('/') ? path(url.slice(0, url.length - 1)) : url
      this.form.host = path(host)
      this.$emit('connect', {
        peer: this.form,
        closeTrigger: this.closeTrigger
      })
    }
```

Reply:
As I said before, I don't think that we should try to cover all the possibilities of input now. I have a task to refactor the whole peer system and this will probably be changed in a near feature.

I can make it recursive but the system should support paths for the peers. The application is not responsible for resolving the URLs.

Reply:
Okay :+1: